src/javascript/app_2/pages/trading/components/elements/list.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const UL = ({ items, className }) =>  (
+    <ul className={`list ${className || ''}`}>
+        { items.map((item, idx) => (
+            <li key={idx} className='list-item'>
+                <span>{item.li}</span>
+            </li>
+        ))}
+    </ul>
+);
```

Comment:
I'm thinking maybe instead of this interface:

```jsx
+                        <UL items={[
+                            { li: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.' },
+                            { li: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.' },
+                            { li: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.' },
+                            { li: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.' },
+                            { li: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.' },
+                        ]} />
```

We can switch to 

```
<UL>
  <LI>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.</LI>
  <LI>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pulvinar id mi ac congue.</LI>
  ...
</U>
```

I `LI` component could just be:

```jsx
const LI = ({children}) => (
   <li className='list-item'>
        <span>{children}</span>
    </li>
);
```